Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -237,6 +307,101 @@ def fetch_incidents(
     return next_run, incidents
 
 
+def set_alert_severity(orca_client: OrcaClient, args: Dict[str, Any]) -> CommandResults:
+    alert_id = args.get("alert_id")
+    score = args.get("score")
+
+    assert alert_id and score
+    demisto.debug(f"Set alert severity {alert_id=} {score=}")
+
+    response = orca_client.set_alert_score(alert_id=alert_id, orca_score=score)
+
+    context = {}
+    if "alert_id" in response:
+        context = {
+            "id": response.get("alert_id"),
+            "details": response.get("details", {}).get("description"),
+            "severity": response.get("details", {}).get("severity")
+        }
+    if "error" in response:
+        context = {
+            "details": response.get("error")
+        }
+
+    return CommandResults(
+        readable_output="Alert severity changed",
+        outputs_prefix="Orca.SetAlertSeverity",
```

Comment:
when updating an entity (like the alert in your case), no need to set an output key for that (SetAlertSeverity), since its not an entity itself, its just a change, so a human readable, a message it was changed and what is the change, should be enough, for the output field, you can use the Orca.Alert key (since its a key) .